cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -47,6 +56,9 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         |** <<administration-security-administration-dbms-custom, Using a custom role to manage DBMS privileges>>
```

Comment:
The different toc seems to not be completely synced for the DBMS privileges, please sync them all with how the section is structured